webpack/app/senaite.core.js

```diff
@@ -111,4 +112,29 @@ document.addEventListener("DOMContentLoaded", () => {
       })
   });
 
+  // Sample view reload when all analyses are in the same state, e.g.
+  // all in to_be_verified or verified state
+  // This is needed to update fields according to the state
+  document.body.addEventListener("listing:submit", (event) => {
+    // get the portal_type from the `data-portal-type` attribute
+    // -> see IBootstrapView
+    let portal_type = document.body.dataset.portalType;
+    // only analyses states propagate up to sample/worksheet
+    if (["AnalysisRequest", "Worksheet"].indexOf(portal_type) == -1) {
```

Comment:
Since the reload relies now on the comparison between the status of the view's context before and after the submit, do we still need this check? In fact, the removal of this conditional would make the logic work in other contexts were the transition of listing items may affect the status of the view's context (thinking about batch or storage atm)

Reply:
Yes, you are right. I think you can remove it.